terraform/20-app/ses.feedback-api.tf

```diff
@@ -0,0 +1,7 @@
+resource "aws_ses_domain_identity" "sender" {
```

Comment:
This is the resource which allows SES to verify that we can send emails from the given domain